src/components/VotedRepoListWrap.tsx

```diff
@@ -40,19 +40,36 @@ const VotedRepoListWrap = (): JSX.Element => {
     setSearchParams({ limit: String(limit + 10) });
   };
 
+  if (!isLoading && !data.length) {
+    return (
+      <div className="mx-auto max-w-7xl px-4 pb-1 mt-10">
```

Comment:
the padding bottom added here looks great but it doesn't exist on the list repositories container; definitely no change when no voted repositories, but could use the padding on the list as well to make it look 💯